cms/templates/cms/welcome.html

```diff
@@ -1,74 +1,89 @@
-            django settings file and haven't added any pages yet.
-        </p>
+<body class="cms-welcome-bg">
+    {% cms_toolbar %}
+    <div class="cms-reset cms-welcome">
+        <header class="cms-welcome-header">
+            <h1 class="cms-welcome-heading">
+                <span class="cms-icon cms-icon-check"></span>
+                {% trans "Installation successful!" %}
+            </h1>
+            <div class="cms-welcome-logo">
+                <span class="cms-hidden">{% trans "django CMS" %}</span>
+            </div>
+        </header>
+        {{ request.version }}
+
+        <section class="cms-welcome-section">
+            <h2>
+                {% if user.is_authenticated %}
+                    2. {% trans "Add your first page" %}
+                {% else %}
+                    1. {% trans "Please login" %}
+                {% endif %}
+            </h2>
+            <p>
+                {% blocktrans %}
+                    Welcome to django CMS version <strong>{{ cms_version }}</strong>.
+                {% endblocktrans %}
+                </p>
+            <p>
+                {% url "admin:index" as admin_index %}
+                {% url "admin:cms_page_add" as admin_add_page %}
+                {% blocktrans %}
+                    You can also login through the <a href="{{ admin_index }}">admin</a> and
+                    <a href="{{ admin_add_page }}?title=Home" class="js-welcome-add">add a page</a> manually
+                    if JavaScript is disabled.
+                {% endblocktrans %}
+        </section>
+
+        <footer class="cms-welcome-footer">
+            <h2 class="cms-hidden">{% trans "Installation Notes" %}</h2>
+            <p class="cms-welcome-links">
+                <a href="//django-cms.org/" target="_blank">{% trans "Website" %}</a>
+                <a href="//django-cms.org/en/support" target="_blank">{% trans "Support" %}</a>
+                <a href="//docs.django-cms.org/" target="_blank">{% trans "Documentation" %}</a>
+            </p>
+            {% blocktrans %}
+                <p class="cms-welcome-notes">If you don't see the django CMS logo at the top, make sure
+                    you linked the <code>static/cms</code> folder to your
+                    static files.</p>
+                <p class="cms-welcome-notes">You're seeing this message because you have
+                    <code>DEBUG = True</code> in your django settings file and
+                    haven't added any pages yet.
+                </p>
+            {% endblocktrans %}
+        </footer>
     </div>
-</div>
-{% render_block "js" %}
-<script type="text/javascript">
-if(window.CMS !== undefined) {
-    CMS.$('#add-page').bind('click', function (e) {
-        e.preventDefault();
-        var modal = new CMS.Modal();
-            modal.open({
-                url: '{% cms_admin_url "cms_page_add" %}?title=Home'
-        });
-    });
-}
-</script>
+    {% render_block "js" %}
 
+    <script>
```

Comment:
might not be the best to use inline js here, but would be a waste to just implement another file?